### DISCUSSION TITLE
Polling Timeout Black Screen Popup

### DISCUSSION THREAD

**Discussion by michaelaguiar** [livewire/livewire#5509] [1. Help]
I have a component that polls the backend every 5 seconds that tells me what page a user is on. When I have 1k+ users on the same page, each polling every 5 seconds, the back end sometimes has a timeout which pops up a black error screen over my page.

Is there a good way to "skip" this poll if there is a timeout and suppress the UI error popup?

**Comment by joshhanley** [comment]
@michaelaguiar not that I know of, the reason is the modal that gets displayed is the error page that Laravel returns if there are any issues.
